### PR TITLE
903604-UG document for .NET MAUI Kanban-Board control 

### PR DESCRIPTION
**Pull request by EzhilarasanElangovan31**
**Description** 

Added the UG document for .NET MAUI Kanban-Board control. I have added the following sections.

- Getting Started
- Column
- Cards
- Workflows
- Localization
- Events
- migration

Updated spline range area images

### CODE REVIEW

File: MAUI/Kanban-Board/Placeholder.md [hunk 1]
```diff
@@ -37,43 +37,43 @@ The following code example describes the above behavior.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Placeholder.md:37]: @EzhilarasanElangovan31  as we are not providing place holder style support in vol 3, remove this md file from toc file

File: MAUI/Kanban-Board/Getting-Started.md [hunk 2]
```diff
@@ -22,31 +22,31 @@ This section provides a quick overview for working with Essential Kanban for .NE
 
 {% highlight xaml %}
 
-    <ContentPage
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Getting-Started.md:25]: @EzhilarasanElangovan31  change md file name as Getting Started (without hyphen)

File: MAUI/Kanban-Board/Getting-Started.md [hunk 3]
```diff
@@ -22,31 +22,31 @@ This section provides a quick overview for working with Essential Kanban for .NE
...
-        . . .    
-        xmlns:kanban="clr-namespace:Syncfusion.Maui.Kanban;assembly=Syncfusion.Maui.Kanban">
-        <Grid>
-            <kanban:SfKanban/>
-        </Grid>
-    </ContentPage>
+<ContentPage
+    . . .    
+    xmlns:kanban="clr-namespace:Syncfusion.Maui.Kanban;assembly=Syncfusion.Maui.Kanban">
+    <Grid>
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Getting-Started.md:28]: @EzhilarasanElangovan31 remove grid control

File: MAUI/Kanban-Board/Column.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ More than one category can be mapped to a column by assigning multiple values to
 
 {% highlight C# %} 
 
-    progressColumn.Categories = new List<object>() { "In Progress", "Validated" };
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Column.md:81]: @EzhilarasanElangovan31 xaml code missing, update it within column, Categories="In Progress,Validated"

File: MAUI/Kanban-Board/Column.md [hunk 5]
```diff
@@ -90,33 +90,33 @@ Header shows the category [`Title`](), items count, min and max informations of
...
-        label.Margin = new Thickness(10); 
-        label.SetBinding(Label.TextProperty, new Binding("Title") );
-        label.TextColor = Color.Purple;
-        label.HorizontalOptions = LayoutOptions.Start; 
-        root.Children.Add(label);
-        return root;
-        });
-    kanban.HeaderTemplate = headerTemplate;
+var headerTemplate = new DataTemplate(() => {
+    StackLayout root = new StackLayout() {
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Column.md:106]: @EzhilarasanElangovan31 code is not started at extreme left, may face alignment issue again

File: MAUI/Kanban-Board/Column.md [hunk 6]
```diff
@@ -227,15 +227,15 @@ The following output demonstrates the above example code.
 
 {% highlight xaml %}
 
-    <kanban:KanbanColumn x:Name="todoColumn" Title="To Do" MinimumLimit="2" MaximumLimit="1">
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Column.md:230]: @EzhilarasanElangovan31 in line 221, update along with its type in APIs like below
image

File: MAUI/Kanban-Board/Column.md [hunk 7]
```diff
@@ -245,20 +245,20 @@ The following output demonstrates the above example code.
...
 
 {% endhighlight %}
 
 {% highlight C# %}
 
-    todoColumn.ErrorBarSettings.Fill = Colors.Green;
-    todoColumn.ErrorBarSettings.MinValidationFill = Colors.Orange;
-    todoColumn.ErrorBarSettings.MaxValidationFill = Colors.Red;
-    todoColumn.ErrorBarSettings.Height = 4;
+todoColumn.ErrorBarSettings.Fill = Colors.Green;
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Column.md:258]: @EzhilarasanElangovan31 when compared with XAML, column code is missing. Update it

File: MAUI/Kanban-Board/Cards.md [hunk 8]
```diff
@@ -21,16 +21,16 @@ The default elements of a card can be customized using the below properties of [
 
 {% highlight C# %}
 
-    new KanbanModel()
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:24]: @EzhilarasanElangovan31 line 19, update as  -> Used to specify the indicator color of a card.

File: MAUI/Kanban-Board/Cards.md [hunk 9]
```diff
@@ -21,16 +21,16 @@ The default elements of a card can be customized using the below properties of [
...
-        Category = "Open",
-        Description = "Analyze customer requirements",
-        IndicatorFill = Colors.Red;
-        Tags = new string[] { "Incident", "Customer" }
-    });
+new KanbanModel()
+{
+    ID = 1,
+    Title = "iOS - 1002",
+    ImageURL = "Image1.png",
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:28]: @EzhilarasanElangovan31 at line 15, refer below, create content and update as notes
image

File: MAUI/Kanban-Board/Cards.md [hunk 10]
```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
...
-            
-            <StackLayout  Orientation="Horizontal"> 
+<DataTemplate>
+    <StackLayout WidthRequest="250" Orientation="Vertical" BackgroundColor="Gray" Padding="10,10,10,10"> 
+        
+        <StackLayout  Orientation="Horizontal"> 
 
-                <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
-                </Label>
+            <Label Text="{Binding Path=Title}" TextColor="Silver" HorizontalOptions="StartAndExpand" >
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:52]: @EzhilarasanElangovan31 we can set Title directly. Is any need to mention as Path=Title?

File: MAUI/Kanban-Board/Cards.md [hunk 11]
```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
...
+        WidthRequest = 250,
+        Orientation = StackOrientation.Vertical,
+        Padding = new Thickness(10),
+        BackgroundColor = Color.Gray
+    };
+
+    StackLayout titleLayout = new StackLayout();
+    Label title = new Label()
+    {
+        TextColor = Color.Silver,
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:87]: @EzhilarasanElangovan31 in maui, we do not have Color.Silver, instead use Colors.Silver

File: MAUI/Kanban-Board/Cards.md [hunk 12]
```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
...
+
+    StackLayout contentLayout = new StackLayout()
+    {
+        Orientation = StackOrientation.Horizontal
+    };
+    Label desc = new Label()
     {
+        WidthRequest = 150,
+        FontSize = 14,
+        TextColor = Color.Silver,
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:101]: @EzhilarasanElangovan31 change here also

File: MAUI/Kanban-Board/Cards.md [hunk 13]
```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
...
+
+    StackLayout root = new StackLayout()
+    {
+        WidthRequest = 250,
+        Orientation = StackOrientation.Vertical,
+        Padding = new Thickness(10),
+        BackgroundColor = Color.Gray
+    };
+
+    StackLayout titleLayout = new StackLayout();
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:84]: @EzhilarasanElangovan31 for titleLayout, orientation as Horizontal missing or using directly HorizontalStackLayout

File: MAUI/Kanban-Board/Cards.md [hunk 14]
```diff
@@ -136,34 +136,34 @@ Create a custom class by inheriting [`DataTemplateSelector`](), and override the
 
 {% highlight c# %}
 
-    public class KanbanTemplateSelector : DataTemplateSelector
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Cards.md:139]: @EzhilarasanElangovan31 at line 129, update DataTemplateSelector link - https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.datatemplateselector?view=net-maui-8.0

File: MAUI/Kanban-Board/Events.md [hunk 15]
```diff
@@ -32,29 +32,29 @@ The default value of the [CardTappedCommandParameter]() is `null`.
...
 
-        private void CardTappedEvent(object args)
-        {
-            //  handle event action.
-        }
+    private void CardTappedEvent(object args)
+    {
+        //  handle event action.
     }
+}
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Events.md:57]: @EzhilarasanElangovan31 line 68 remove for now, not planned to provide support in vol 3

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 need to include localization md file and its content

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 modify as Kanban in migration md file
image

File: MAUI/Kanban-Board/Placeholder.md (hunk 1 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Placeholder.md:37]: removeed this md file from toc file

File: MAUI/Kanban-Board/Getting-Started.md (hunk 3 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Getting-Started.md]: removed grid control

File: MAUI/Kanban-Board/Column.md [hunk 16]
```diff
@@ -38,35 +38,35 @@ You can also define the exact column width using [`ColumnWidth`]() property of [
...
+kanban.ColumnWidth = 250;
 
 {% endhighlight %}
 
 {% endtabs %}
 
 ## Categorizing Columns
 
-If [`ItemsSource`]() contains custom objects, the path of the property which can be used to categorize the card should be explicitly defined using [`ColumnMappingPath`]() property. By default, [`SfKanban`]() will automatically categorize the items using [`Category`]() property of [`KanbanModel`]().
+To categorize columns based on a specific property, you must explicitly define the property path using the [`ColumnMappingPath`]() property. However, only the properties of [`KanbanModel`]() can be assigned to [`ColumnMappingPath`](). By default, [`SfKanban`]() will categorize the items using the Category property of KanbanModel."
```
[the remark this answers is not in the document]
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Column.md]: removed

File: MAUI/Kanban-Board/Cards.md (hunk 8 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md:24]: Updated

File: MAUI/Kanban-Board/Cards.md (hunk 9 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md:30]: Added a note

File: MAUI/Kanban-Board/Cards.md (hunk 14 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md:139]: Updated the link

File: MAUI/Kanban-Board/Column.md (hunk 4 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Column.md:81]: Added xaml code

File: MAUI/Kanban-Board/Column.md (hunk 5 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Column.md:116]: aligned the code

File: MAUI/Kanban-Board/Column.md (hunk 6 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Column.md:230]: Updated

File: MAUI/Kanban-Board/Getting-Started.md (hunk 2 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Getting-Started.md:25]: Changed

File: MAUI/Kanban-Board/Cards.md (hunk 10 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md]: Updated

File: MAUI/Kanban-Board/Cards.md (hunk 11 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md]: Updated

File: MAUI/Kanban-Board/Cards.md [hunk 17]
```diff
@@ -42,83 +42,83 @@ You can replace the entire card template with your own design using [`CardTempla
...
+        Orientation = StackOrientation.Vertical,
+        Padding = new Thickness(10),
+        BackgroundColor = Color.Gray
+    };
+
+    StackLayout titleLayout = new StackLayout();
+    Label title = new Label()
+    {
+        TextColor = Color.Silver,
+        HorizontalOptions = LayoutOptions.StartAndExpand
```
[the remark this answers is not in the document]
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md]: Updated

File: MAUI/Kanban-Board/Cards.md (hunk 13 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md]: Updated

File: MAUI/Kanban-Board/Cards.md (hunk 12 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Cards.md]: Updated

File: MAUI/Kanban-Board/Localization.md [hunk 18]
```diff
@@ -0,0 +1,65 @@
...
+
+using Syncfusion.Maui.Kanban;
+using System.Globalization;
+using System.Resources;
+
+public partial class App : Application
+{
+	public App()
+	{
+		 InitializeComponent();
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Localization.md:30]: @EzhilarasanElangovan31 code alignment not proper, modify it

File: MAUI/Kanban-Board/Localization.md [hunk 19]
```diff
@@ -0,0 +1,65 @@
...
+         CultureInfo.CurrentUICulture = new CultureInfo("fr-FR");
+         var ResXPath = "KanbanSample.Resources.SfKanban";
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/Localization.md:32]: @EzhilarasanElangovan31 refer chart, similar to it add commented lines for better readability

File: MAUI/Kanban-Board/migration.md [hunk 20]
```diff
@@ -111,39 +111,11 @@ The following table illustrates the API migration for the kanban.
...
-<tr>
-<td>SelectedBorderColor</td>
-<td>SelectedStroke</td>
-</tr>
-<tr>
-<td>SelectedBorderThickness</td>
-<td>SelectedStrokeWidth</td>
-</tr>
-</table>
+* Support for keeping cards.
```
Comment (VimalaThirumalaiKumarSF3739) [inline MAUI/Kanban-Board/migration.md:116]: @EzhilarasanElangovan31 modify as - Support for keeping cards in the source column location during dragging until dropped.

**Comment by VimalaThirumalaiKumarSF3739** [comment]
@EzhilarasanElangovan31 in PR template - no updated details for spline range area. Also, in description, you have mentioned overview files, but not have changes in that file. Do not copy paste the PR template from old PRs blindly

File: MAUI/Kanban-Board/Localization.md (hunk 18 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Localization.md]: Updated alignment

File: MAUI/Kanban-Board/Localization.md (hunk 19 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/Localization.md]: add comment lines

File: MAUI/Kanban-Board/migration.md (hunk 20 above)
Comment (EzhilarasanElangovan31) [reply MAUI/Kanban-Board/migration.md]: Modified

**Comment by VimalaThirumalaiKumarSF3739** [comment]
Fine

**Review by Saravanan-Madhesh — APPROVED** [review]

**Pull request merged**